package dev.alexeev.user_service.dto.user;

import dev.alexeev.user_service.dto.card.PaymentCardResponseDto;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
public class UserWithCardsResponseDto implements Serializable {
  private Long id;
  private String name;
  private String surname;
  private LocalDate birthDate;
  private String email;
  private boolean active;
  private LocalDateTime createdAt;
  private LocalDateTime updatedAt;
  private List<PaymentCardResponseDto> cards;
}